cycode/cli/commands/scan/code_scanner.py

```diff
@@ -312,17 +312,36 @@ def scan_documents(
         )
         return
 
-    scan_batch_thread_func = _get_scan_documents_thread_func(context, is_git_diff, is_commit_range, scan_parameters)
+    scan_batch_thread_func, aggregation_id = _get_scan_documents_thread_func(
+        context, is_git_diff, is_commit_range, scan_parameters
+    )
     errors, local_scan_results = run_parallel_batched_scan(
         scan_batch_thread_func, documents_to_scan, progress_bar=progress_bar
     )
+    aggregation_report_url = _try_get_aggregation_report_url_if_needed(
+        scan_parameters, context.obj['client'], context.obj['scan_type'], aggregation_id
+    )
 
     progress_bar.set_section_length(ScanProgressBarSection.GENERATE_REPORT, 1)
     progress_bar.update(ScanProgressBarSection.GENERATE_REPORT)
     progress_bar.stop()
 
     set_issue_detected_by_scan_results(context, local_scan_results)
-    print_results(context, local_scan_results, errors)
+    print_results(context, local_scan_results, errors, aggregation_report_url)
```

Comment:
pls add `aggregation_report_url` field to `LocalScanResult` class instead of an additional arg

Reply:
LocalScanResult is relevant per scan ID and the aggregation_report_url is relevant for all the scans together

Reply:
agree. don't we want to pass it to context? since it's global for all scans. and we already provide context to all printers

Reply:
@saramontif wdyt?